utils/frame/sub-du/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "sub-du"
+version = "0.1.0"
+authors = ["kianenigma <kian.peymani@gmail.com>"]
+edition = "2021"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0" }
+async-std = { version = "1.12.0", features = ["attributes"] }
+jsonrpsee = { version = "0.15.1", features = ["jsonrpsee-types", "jsonrpsee-http-client", "jsonrpsee-ws-client"] }
+separator = "0.4.1"
+ansi_term = "0.12.1"
+env_logger = "0.9"
```

Comment:
switch this to `tracing-subscriber` to get logs from jsonrpsee as well if you want that :P